web/themes/custom/pineapple/sass/components/_card.scss

```diff
@@ -11,6 +12,11 @@
 
     .card-image-container {
       opacity: 0.9;
+      transition: 0.5s;
+    }
+
+    .btn--underlined {
+      background-color: darken($light-gray, 5%);
     }
```

Comment:
What's the point of this?

Reply:
The button has a white background without it on the user cards.

Reply:
Why does the button have a white background to begin with?

Reply:
I looked into it and it was because the link on the clickable card wasn't working and for some reason that caused the button and the title of the card to have different styling. But it's fixed now and the sass code is removed.